tests/e2e/config/jest.setup.js

```diff
@@ -1,18 +1,16 @@
 /**
  * External dependencies
  */
-import { get } from 'lodash';
 import {
 	enablePageDialogAccept,
-	isOfflineMode,
 	setBrowserViewport,
 	switchUserToAdmin,
 	switchUserToTest,
 	visitAdminPage,
 } from '@wordpress/e2e-test-utils';
 
 // Set the default test timeout.
-let jestTimeoutInMilliSeconds = 40000;
+let jestTimeoutInMilliSeconds = 60000;
```

Comment:
Waiting 60 seconds for a timeout seems excessive, anything that is logical and takes more than 40 seconds?

Reply:
I actually wanted to make it bigger & make sure that if the timeout expires then the element isn't really found on the page. This setting would eliminate the slow UI as a reason and highlight that there is an issue with the page. As this only affects failing tests I don't think the impact is that big. We could try it out and see the outcome.